src/main/java/mate/controller/CarCreationController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+public class CarCreationController extends HttpServlet {

```

Comment:
```suggestion
public class AddCarController extends HttpServlet {

```

you have controller AddDriverToCarController, let's follow the same pattern everywhere

Reply:
make as was suggested, the same for other classes with similar names